### ISSUE TITLE
IE8 fallback not added if IE9 or 10 are also in the browser option

### ISSUE THREAD

**Issue by bob2021** [robwierzbowski/node-pixrem#54]
If I apply pixrem to this css:

`a { font-size: 0.75rem; }`

I would expect a fallback to be added for IE >= 8, but not IE >= 9.  

If I specify 'ie 8' in the browser option then everything works ok, but if I specify 'ie >= 8' then the fallback is not added.

**Comment by iamvdo** [comment]
Could not reproduce.
`ie 8` or `ie >= 8` produce same result.

Be sure to check your workflow (minifer, etc.)

**Comment by bob2021** [comment]
I have double checked but I am still getting different results - here's the script I'm using to test:

```javascript
'use strict';
var fs      = require('fs');
var pixrem  = require('pixrem');
var postcss = require('postcss');

var css = 'h1 { font-size: 1rem; }';
var processedCss = '';

processedCss = postcss([pixrem({
    browsers: 'ie >= 8'
})]).process(css).css;
console.log(processedCss);

processedCss = postcss([pixrem({
    browsers: 'ie 8'
})]).process(css).css;
console.log(processedCss);
```

If you could take another look or point out what I'm doing wrong it would be much appreciated.  I'm using pixrem version 3.0.0

Thanks

**Comment by iamvdo** [comment]
Sorry, there was a bug. Fixed and released as 3.0.1
Thanks :+1:
